src/frontend/defaultBrowserWindowOptions.ts

```diff
@@ -11,7 +11,8 @@ export default function (flowrStore: Store): BrowserWindowConstructorOptions {
     minWidth: 430,
     minHeight: 270,
     title: 'FlowR',
-    fullscreen: kiosk,
+    fullscreen: true,
+    kiosk,
```

Comment:
This change will need to be tested and validated on every OS as this will modify the behaviour the users are accustomed to, and the two options (fullscreen and kiosk) have specific behaviours depending on the OS on which electron is running.

Reply:
On macos are working as they are supposed to work, but I don't have a windows pc